src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ViewCellRenderer.cs

```diff
@@ -268,8 +268,16 @@ public void DisconnectHandler()
 
 			public override void AddView(AView child)
 			{
-				base.AddView(child);
-				_currentView = child;
+				if (child.Parent is WrapperView wrapperView)
```

Comment:
With shadows, the View is wrapped with a container, WrapperView, and for that reason the view already has a Parent.